app/controllers/experiments_controller.rb

```diff
@@ -68,12 +68,10 @@ def update
     if @experiment.save
       flash[:success] = t('experiments.update.success_flash',
                           experiment: @experiment.name)
-
-      redirect_to project_path(@experiment.project)
     else
       flash[:alert] = t('experiments.update.error_flash')
-      redirect_to :back
     end
+    redirect_to :back
```

Comment:
I'd rather use helper for experiment path: `canvas_experiment_path(@experiment)`, since `:back` is a bit hackish.